substrate/frame/revive/uapi/src/host.rs

```diff
@@ -244,11 +244,7 @@ pub trait HostFn: private::Sealed {
 	///
 	/// - `addr`: The address of the contract.
 	/// - `output`: A reference to the output data buffer to write the code hash.
-	///
```

Comment:
It would be nice to instead explain the returned value here :)

Reply:
maybe not here, this function explains itself. We can add more documentation on implementation level

Reply:
add documentation on implementation level

Reply:
Documentation on there doesn't hurt. But why not accurately document the API?

Reply:
I can add it here as well but I think this is documented well, it takes an address and reference to write output.

Reply:
Yeah please do. We don't want people to needlessly puzzle together the full picture of our API. The error was documented before.

Reply:
Yes sure it writes output. Now the error is gone from the method signature. How are people supposed to know _what_ output is written when the address was not a contract or doesn't exist?

Reply:
fair enough